components/vocabularies/ui/src/main/resources/PhenoTips/IndexOntology.xml

```diff
@@ -60,14 +60,19 @@
 #end
 #if ("$!{request.ajax}" != '1')
 $xwiki.jsx.use('PhenoTips.IndexOntology')##
+$xwiki.jsx.use('PhenoTips.Widgets')##
+$xwiki.ssx.use('PhenoTips.Widgets')##
 == $services.localization.render('phenotips.indexOntology.heading') ==
 (% class="half" %)(((
 {{html wiki=false clean=false}}
 #foreach ($ontology in ['hpo', 'omim', 'chebi', 'hgnc'])
 &lt;form class="xform solr-admin" action='' method="post"&gt;
 &lt;dl&gt;
 &lt;dt&gt;
-&lt;label for="ontology-${ontology}-url"&gt;$services.localization.render("phenotips.IndexOntology.${ontology}")&lt;/label&gt;
+&lt;label style="display:inline" for="ontology-${ontology}-url"&gt;$services.localization.render("phenotips.IndexOntology.${ontology}")&lt;/label&gt;
```

Comment:
The label on-click behaviour then interferes with the tooltip (it opens and then immediately closes).